### ISSUE TITLE
Question: should we create a "sandbox-mode" activity ?

### ISSUE THREAD

**Issue by moreau-nicolas** [openwebnet/openwebnet-android#40]
In #39 I said that it would be nice to have a "sandbox-mode" Activity where users (*and ourselves obviously*) could try things and see live results.

It could be hidden by default and enabled in the settings. Perhaps this sandbox activity could have a "save as card" feature ? (with better wording though)

What do you think about it ?

**Comment by niqdev** [comment]
I never thought about "device card" in this way, this could be like a console for advanced user?
I like the idea to transform the req/res to a card!
Anyway I wouldn't add it as preference, but as an item in the navigation drawer?!

**Comment by moreau-nicolas** [comment]
Yes, I was thinking of adding a "sandbox mode" item in the drawer, but having a preference toggling its visibility. Could be the same as "enable debug mode".

**Comment by niqdev** [comment]
Is related to the [fab issue](https://github.com/openwebnet/openwebnet-android/issues/40), if we have a separated "sandbox activity", do we really still have the need of the icon DEBUG in ActionMenu and also the hiding toggle for device cards?
I was thinking about your "Sanbox" just as an item menu between "Add environment" and "Settings"
